packages/react-core/src/demos/MasterDetailDemo.md

```diff
@@ -1571,19 +1579,23 @@ class MasterDetailCardView extends React.Component {
 
     const toolbarItems = (
       <React.Fragment>
-        <ToolbarItem>{this.buildFilterDropdown()}</ToolbarItem>
-        <ToolbarItem>
-          <Button variant="primary">Create a Project</Button>
-        </ToolbarItem>
-        <ToolbarItem>
-          <Dropdown
-            onSelect={this.onToolbarKebabDropdownSelect}
-            toggle={<KebabToggle onToggle={this.onToolbarKebabDropdownToggle} id="card-view-data-toolbar-dropdown" />}
-            isOpen={isLowerToolbarKebabDropdownOpen}
-            isPlain
-            dropdownItems={toolbarKebabDropdownItems}
-          />
-        </ToolbarItem>
+        <ToolbarToggleGroup toggleIcon={<FilterIcon />} breakpoint="xl">{this.buildFilterDropdown()}</ToolbarToggleGroup>
```

Comment:
In this example, the select doesn't need to be `hidden/expanded`, so you don't need to use a `toggle-group` to hide filters.